analyzeme/src/profiling_data.rs

```diff
@@ -174,6 +174,19 @@ impl ProfilingData {
             thread_id: raw_event.thread_id,
         }
     }
+
+    // This method is added only in order to make it easier to fix
+    // https://github.com/rust-lang/measureme/issues/173 in the future.
+    //
+    // It is marked as "internal" because we want to avoid that downstream tooling
+    // depends on RawEvent directly. Do not use this method directly.
```

Comment:
Hmm this makes me a bit worried. If we don't strictly needed it right perhaps we should just not include it in this release?

Reply:
I don't have a strong opinion about this. I can remove it if you prefer.

Reply:
Removed